### PR TITLE
Feature/continous integration

### PR DESCRIPTION
**Pull request by duongdang**
The main goal of this PR is to able to run test on travis-ci. The main changes include:
* Use a delivered version of extraction-framework instead of a snapshot
* Create a local dump so that extraction tests can be run in travis-ci
* Basic travis-ci script.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# branches:
+#   only:
+#     - master
+before_install:
+  - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
+script: "mvn test"
+notifications:
+  email:
+    recipients:
+      - dang.duong@gmail.com
```
Comment (jimkont) [inline .travis.yml]: Can you change this to "riteshoneinamillion@gmail.com" for now?

Comment (duongdang) [inline .travis.yml]: Done.

**Comment by jimkont** [comment]
+1, thanks a lot @duongdang

**Comment by duongdang** [comment]
@jimkont Thanks for merging. 

Just to complete the PR, it might be great to add the build to travis-ci.org/dbpedia and add the build badge to README.md.

**Pull request merged**